hammr/commands/image/image.py

```diff
@@ -237,8 +237,8 @@ def do_deploy(self, args):
                 printer.out("Published image with name '" + pimage.name + " cannot be deployed", printer.ERROR)
                 return 2
 
-            if self.is_targeted_cloud_openstack(pimage):
-                printer.out("Deployments for this cloud are not supported by hammr yet.", printer.ERROR)
+            if not self.is_targeted_cloud_amazon(pimage):
+                printer.out("Hammr only supports deployments for Amazon AWS.", printer.ERROR)
```

Comment:
Amazon AWS is a small duplication (Amazon Amazon Web Service) but I think it's ok (have the same pb with Azure ARM)